static/data-ingestion.yaml

```
@@ -1067,23 +671,17 @@ paths:
             schema:
               type: array
               items:
-                $ref: '#/components/schemas/FeedProductOverride'
+                $ref: '#/components/schemas/FeedPricebook'
             examples:
-              ProductPermissionsPerCustomerGroup:
-                summary: Set product permission for customer group
-                description: Hide product prices and add product to Cart for "demo-product" in the "us" website and the "hide-price" customer group
+              FeedWithPricebookInformation:
+                summary: Example of product price books feed to update
```

```suggestion
                summary: Update Price Book "dealer-north" currency to "EUR"
```

Suggest to remove name - otherwise it look the same as "PUT"

---

Updated. 
Why remove name? we still have same priceBookId

---

Changed summary, but left changed name - we still have save price book id, so example shows that we can change name as well